src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -149,62 +134,59 @@ const TableHeader: React.FC<TableHeaderProps> = ({
     };
   }, [leaveSeat]);
 
-  const updateTableTitle = useCallback(
-    async (newTableTitle) => {
+  const [updateResponse, updateTables] = useAsyncFn(
+    async (values: EditTableForm) => await updateTableInfo(values),
+    []
+  );
+
+  const updateTableInfo = useCallback(
+    async (values: EditTableForm) => {
       if (!user) return;
 
       const venueTables = tables.map((table) => {
         if (table.reference === tableOfUser?.reference) {
-          return { ...table, title: newTableTitle };
+          return {
+            ...table,
+            title: values.title,
+            subtitle: values.description,
+            capacity: values.capacity,
+          };
         }
         return table;
       });
 
       // Ideally we want to do this only when config.tables doesn't exist. Otherwise we want to update only a single table.
-      await updateVenue_v2(
+      return await updateVenue_v2(
         { name: venueName, tables: venueTables },
         user
-      ).catch(() => {
-        // @debt Replace generic error when proper error handling is added
-        setError(GENERIC_ERROR);
-
-        const msg = "[updateVenue_v2] updating tables in venue.config.tables";
-        const context = {
-          location: "molecules::TableHeader::updateTableTitle",
-        };
-
-        console.warn(msg, context);
-        Bugsnag.notify(msg, (event) => {
-          event.severity = "warning";
-          event.addMetadata("context", context);
-        });
-      });
+      );
     },
     [tableOfUser?.reference, tables, user, venueName]
   );
 
   return (
     <div className="row table-header">
       <div className="table-header__leave-table">
-        <button
-          type="button"
-          title={"Leave " + seatedAtTable}
-          className="table-header__leave-button"
-          id="leave-seat"
-          onClick={leaveSeat}
-        >
-          <FontAwesomeIcon
-            className="table-header__leave-button--icon"
-            icon={faChevronLeft}
-            size="xs"
-          />
-          Leave table
-        </button>
+        <div>
+          <button
+            type="button"
+            title={"Leave " + seatedAtTable}
```

Comment:
[nit]
```suggestion
            title=`Leave ${seatedAtTable}`
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a